timereport/app.py

```diff
@@ -54,10 +54,25 @@ def index():
         slack_responder(response_url, "Add action OK")
 
     if action == "list":
-
         # python-backend
-        response = get_user_by_id(f'{python_backend_url}/user', payload.get('user_id'))
-        # post back to slack as a code formatted output ``` data ```
-        for r in response:
+
+        # get everything
+        get_by_user = get_user_by_id(f'{python_backend_url}/user', payload.get('user_id'))
+        for r in get_by_user:
             slack_responder(response_url, f'```{str(r)}```')
```

Comment:
What does this do?

Reply:
it will basically just post to slack response_url with the clasic ``` stuff here ```formatting